docs/migrations.md

```diff
@@ -113,7 +113,7 @@ If learning via a video if more your style, we have a short recording explaining
 
 ## Keeping Track of Migrations
 
-When our CI/CD process runs the migration process, it keeps track of which migration scripts have ran inside our `efcms-deploy-$ENV` table using the `./web-api/track-successful-migrations.js` node script.  This script looks at the migration directory and writes entries into dynamo to track the migrations that have ran.  An example of one of these records looks like this:
+When our CI/CD process runs the migration process, it keeps track of which migration scripts have ran inside our `efcms-deploy-$ENV` table using the `./web-api/track-successful-migrations.ts` node script.  This script looks at the migration directory and writes entries into dynamo to track the migrations that have ran.  An example of one of these records looks like this:
```

Comment:
Nit: it keeps track of which migration scripts have ran inside our `efcms-<ENV>-<VERSION>` table